lms/djangoapps/courseware/tests/test_access.py

```diff
@@ -301,9 +301,9 @@ def test_has_access_in_preview_mode_with_group(self):
         """
         Test that a user masquerading as a member of a group sees appropriate content in preview mode.
         """
-        partition_id = 0
-        group_0_id = 0
-        group_1_id = 1
+        partition_id = 100
```

Comment:
Here's another case of using the magic 100 value.  Consider adding a constant for this somewhere so we can use it in all affected tests - and the comment to explain why would be in that single place where the constant is defined.